backend/models/core/Entity.js

```diff
@@ -363,8 +383,8 @@ class Entity {
 
     if (nonExistingVal.length) {
       const insertValuesBatchSQL = `INSERT INTO \`values\`(entity_id, attribute_id,
-      value_string, value_long_string, value_double  
-    ) VALUES (:entity_id, :attribute_id, :value_string, :value_long_string, :value_double) 
+      value_string, value_long_string, value_double, value_boolean  
+    ) VALUES (:entity_id, :attribute_id, :value_string, :value_long_string, :value_double, :value_boolean) 
```

Comment:
1. MySQL has no boolean type - MySQL handles boolean as an integer
2. However, I realized that we do not have an integer data type in the Entity model! Can you kindly put a  `value_integer` here, with `INT(11)` type?

Reply:
On second thought, never mind this :)